android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -139,36 +139,39 @@ public boolean onItemLongClick(AdapterView<?> av, View v, int position, long id)
    */
   private class DiscoveryServiceConnection implements ServiceConnection {
     private PwoDiscoveryService mDiscoveryService;
+    private boolean mRequestOldPwos;
```

Comment:
I know this is not completely defined yet, but what are we considering to be old pwos? Those discovered on the most recent scan?